docs/content/workers/api-worker.md

```diff
@@ -2,33 +2,33 @@
 [//]: # (TODO - find out whether WorkerSystem is ECS and how it fits into a generic workflow.)
 # Workers: API - Worker
 
-_This document relates to both *[GameObject-MonoBehaviour and  ECS workflows](../intro-workflows-spos-entities.md)_
+_This document relates to both *[GameObject-MonoBehaviour and  ECS workflows]({{urlRoot}}/content/intro-workflows-spos-entities.md)_
 
-Before reading this document, see the documentation on [workers in the GDK](workers-in-the-gdk.md)) and [Connecting to the SpatialOS Runtime](connecting-to-spos.md).
+Before reading this document, see the documentation on [workers in the GDK]({{urlRoot}}/content/workers/workers-in-the-gdk.md)) and [Connecting to the SpatialOS Runtime]({{urlRoot}}/content/connecting-to-spos.md).
 
 We provide the `Worker` class to bootstrap the creation of your workers.
 
-Note that the connection between your game and the SpatialOS [Runtime](../glossary.md#spatialos-runtime), depends on the successful creation of your workers.  During their creation, workers attempt to connect to the SpatialOS Runtime. Only a successful connection leads to the creation of a worker.
+Note that the connection between your game and the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime), depends on the successful creation of your workers.  During their creation, workers attempt to connect to the SpatialOS Runtime. Only a successful connection leads to the creation of a worker.
 
-Upon successfully connecting to the SpatialOS Runtime and creating your worker and its corresponding [ECS world](../glossary.md#unity-ecs-world), the following systems are added by the `Worker` class your [worker’s world](../glossary.md#workers-world). These systems  ensure that any changes in any SpatialOS entity is correctly synchronized between the SpatialOS [Runtime](../glossary.md#spatialos-runtime) and the [worker's view](../glossary.md#workers-view).
+Upon successfully connecting to the SpatialOS Runtime and creating your worker and its corresponding [ECS world]({{urlRoot}}/content/glossary.md#unity-ecs-world), the following systems are added by the `Worker` class your [worker’s world]({{urlRoot}}/content/glossary.md#worker-s-world). These systems  ensure that any changes in any SpatialOS entity is correctly synchronized between the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime) and the [worker's view]({{urlRoot}}/content/glossary.md#worker-s-view).
 
 ## Systems ensuring synchronization between the Runtime and workers
```

Comment:
uhh didn't see that before. will have to go in like this for now, but not all of them are strictly speaking responsible for synchronization